#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("canny_threshold", double_t, 0, "Canny Edge Detection Threshold", 220, 0, 255)
gen.add("canny_multiplier", double_t, 0, "Canny Edge Detection Multiplier", 3, 1, 5)

exit(gen.generate("mcr_contour_matching", "contour_finder", "ContourFinder"))
